config/local_example.txt

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
What is the purpose of the local_example.txt file? Is a local config necessary or is it covered by default.json?

I think this should be moved to local.json, or local.json.example, and it should be valid json. If you want to annoate with comments, keep the comments in a js style block comment at the top of the file (even though this is not valid json, I think it is an okay compromise).